stl/inc/xstring

```diff
@@ -1825,11 +1842,9 @@ using wstring_view   = basic_string_view<wchar_t>;
 
 
 template <class _Elem, class _Traits>

```

Comment:
We should remove all occurrences of `_Traits` from this template, as well as the `is_same_v<_Traits, ...` constraint, and let the default template argument handle the traits type for us.